cleaning_services/data/worksheet_template.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="fsm_worksheet_template" model="worksheet.template" context="{'worksheet_no_generation': True}" forcecreate="1">
```

Comment:
Same as other worksheet related records, this is automatically created, no need to define (or assign in project). You can remove this file.